python/ray/util/spark/__init__.py

```diff
@@ -2,6 +2,12 @@
     setup_ray_cluster,
     shutdown_ray_cluster,
     MAX_NUM_WORKER_NODES,
+    serve_global_ray_cluster,
 )
 
-__all__ = ["setup_ray_cluster", "shutdown_ray_cluster", "MAX_NUM_WORKER_NODES"]
+__all__ = [
+    "setup_ray_cluster",
+    "shutdown_ray_cluster",
+    "MAX_NUM_WORKER_NODES",
+    "serve_global_ray_cluster",
```

Comment:
just to be consistent, should we name it `setup_global_ray_cluster`.

Actually should we just have `setup_ray_cluster` with an extra parameter called `is_global`?